frontend/src/metabase/components/CollapseSection/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./CollapseSection";
```

Comment:
🤔 I haven't been doing this, I've been importing from URIs with dir and filename like `path/to/CollapseSection/CollapseSection`

We've touched on this in a RFC.

I don't know, I just think it's one fewer file, and generally less indirection.
But happy to follow any of the patterns.